frontend/src/components/managePermissions/ManagePermissions.vue

```diff
@@ -54,18 +54,63 @@ const props = defineProps({
         type: Array as PropType<FamAccessControlPrivilegeGetResponse[]>,
         default: [],
     },
+    newUserInTable: {
+        type: String,
+        default: '',
+    },
 });
 
+type UserRoleOrAppAdminOrAccessControlPrivilege =
+    | FamApplicationUserRoleAssignmentGet
+    | FamAppAdminGetResponse
+    | FamAccessControlPrivilegeGetResponse;
+
+type UserRoleOrAppAdminOrAccessControlPrivilegeWithIsNew =
+    UserRoleOrAppAdminOrAccessControlPrivilege & {
+        isNewUser: boolean;
+    };
+const newUserInTable = props.newUserInTable.split(',').map(Number);
```

Comment:
This `newUserInTable` being used among all `userRoleAssignments`, `applicationAdmins` and `delegatedAdmins` 3 ref will have bad consequences. See the below code that you are trying to identify the `isNewUser`.
Are you able to identify the `newUserInTable` is really for just `userRoleAssignments` or just for `applicationAdmins` or just for `delegatedAdmins`, or for ALL of them?

Even though we currently display ONE TAB at a time, this really will confuse other developers (that might be the least bad thing) or worse that other developers may not even notice (and having headache debugging what goes wrong later)  because this bug is very hidden.

Reply:
that makes sense, I'll refactor it to avoid that